ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -110,7 +111,13 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
               SystemClock.nanoTime(),
               createWebViewEvent(webView, url)));
     }
-
+    @Override
+    public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        ReactContext reactContext = (ReactContext) view.getContext();        
+        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(url)); 
```

Comment:
Please check if it is http or https before handling.

Also does `startActivity` work without `FLAG_NEW_TASK`